### PR TITLE
FF-141 fixing deployment with proxy.

### PR DESCRIPTION
**Pull request by open-schnick**

### CODE REVIEW

File: src/main/resources/application-prod.properties [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
 spring.data.mongodb.host=${DB_CONTAINER_NAME}
 spring.data.mongodb.username=${DB_USERNAME}
 spring.data.mongodb.password=${DB_PASSWORD}
-#----------------- Custom ----------------------
-filefighter.disable-password-check=false
\ No newline at end of file
+#----------------- Custom --------------------------
+filefighter.disable-password-check=false
+#----------------- ReverseProxyOnly ----------------
+server.servlet.context-path=/api/
```
Comment (qvalentin) [inline src/main/resources/application-prod.properties:9]: where is this being used?

**Review by qvalentin — COMMENTED** [review]
did you also change /health?

File: src/main/resources/application-prod.properties (hunk 1 above)
Comment (open-schnick) [reply src/main/resources/application-prod.properties:9]: that is the base url. -> on e.g `/` an error appears, but that cannot be reached from proxy

**Comment by open-schnick** [comment]
> did you also change /health?

Well i basicly changed everything except health.

**Review by qvalentin — APPROVED** [review]
so on the rest server it is now /v1/user and still /health?

Works on prod, so just merge it.

**Comment by open-schnick** [comment]
> so on the rest server it is now /v1/user and still /health?
> 
> Works on prod, so just merge it.

yes. exactly

**Comment by open-schnick** [comment]
> > so on the rest server it is now /v1/user and still /health?
> > Works on prod, so just merge it.
> 
> yes. exactly

wait nvm. the server without proxy listens on `/api/v1/user...` or `/api/health` but with proxy it works as intended now. i didnt check swagger locally.

**Comment by qvalentin** [comment]
> > > so on the rest server it is now /v1/user and still /health?
> > > Works on prod, so just merge it.
> > 
> > 
> > yes. exactly
> 
> wait nvm. the server without proxy listens on `/api/v1/user...` or `/api/health` but with proxy it works as intended now. i didnt check swagger locally.

thats because of what we changed on the ngix config yesterday

**Comment by open-schnick** [comment]
> > > so on the rest server it is now /v1/user and still /health?
> > > Works on prod, so just merge it.
> > 
> > 
> > yes. exactly
> 
> wait nvm. the server without proxy listens on `/api/v1/user...` or `/api/health` but with proxy it works as intended now. i didnt check swagger locally.

i am actually even smarter then i remembered xD. 
Locally -> on `dev` nothing has changed except the requests go to `/v1/users` instead of `/api/v1/users`

**Pull request merged**